### ISSUE TITLE
App goes to sleep and stops recording

### ISSUE THREAD

**Issue by casperbakker** [openstreetcam/ios#5]
I had the app open and recording, with iOS 10 beta 7 on an iPhone 6 and connected to power, when the phone goes into sleep mode. The recording also stopped when the phone got into sleep mode.

A lot of apps, like Google Maps, keeps the phone from going into sleep mode when in navigating mode. Would be a welcome addition to this app to keep it from sleeping while recording.

**Comment by bogdans-telenav** [comment]
@casperbakker thanks for reporting!
We call the API to keep the phone from going to sleep (because we can not take photos while the app is in background) but there is a bug there we will have to investigate more. It is quite annoying because I could reproduce this only once on iOS 9.3.3 a few weeks ago. I will install iOS 10 beta and try to reproduce this.

**Comment by bogdans-telenav** [comment]
@casperbakker i believe this issue is no longer reproducible on the recording screen. I will now close this since i did not experienced it any more since then. If you are still reproducing this please open another issue.
